### PR TITLE
Fix up code view color scheme

### PR DESCRIPTION
**Pull request by ptomato**
https://phabricator.endlessm.com/T24352

### CODE REVIEW

File: data/application.css [hunk 1]
```diff
@@ -60,6 +60,12 @@ toolbox headerbar {
     like the text was meant to be 60% opaque. Instead I've multiplied the
     background alpha by 0.6 */
     color: #fdfdfd;
+    font-size: 18px;
+}
+
+.sourceview .left {
+    background: rgba(6, 20, 39, 0.80);
```
Comment (tchx84) [inline data/application.css]: Looking at the reference image, I think you meant:
```
.sourceview border {
   background-color: rgba(6, 20, 39, 0.80);
}
```

Comment (tchx84) [reply data/application.css]: Never-mind is the same thing it appears!

File: com.endlessm.HackToolbox.json.in [hunk 2]
```diff
@@ -43,6 +43,20 @@
         }
       ]
     },
+    {
+      "name": "Hack-font",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D -m644 *.ttf -t /app/share/fonts"
```
Comment (tchx84) [inline com.endlessm.HackToolbox.json.in:50]: lesson learn from knowledge apps, we need to `"post-install": ["fc-cache -f /app/share/fonts/`"]`

**Review by tchx84 — CHANGES_REQUESTED** [review]
Just one quick comment.

File: com.endlessm.HackToolbox.json.in (hunk 2 above)
Comment (ptomato) [reply com.endlessm.HackToolbox.json.in:50]: huh, it worked without that! But I'll add it

**Pull request merged**